### PR TITLE
WhiteMagic.JoystickGremlin version 13.3

### PR DESCRIPTION
**Pull request by iskela45**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15815)

### CODE REVIEW

File: manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: WhiteMagic.JoystickGremlin
+PackageVersion: 13.3
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.installer.yaml:7]:
```suggestion
  Architecture: x86
```

File: manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: WhiteMagic.JoystickGremlin
+PackageVersion: 13.3
```
Comment (ItzLevvie) [inline manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.installer.yaml:4]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: WhiteMagic.JoystickGremlin
+PackageVersion: 13.3
```
Comment (ItzLevvie) [inline manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: WhiteMagic.JoystickGremlin
+PackageVersion: 13.3
```
Comment (ItzLevvie) [inline manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.yaml:4]:
```suggestion
PackageVersion: 13.1.0
```

File: manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: WhiteMagic.JoystickGremlin
+PackageVersion: 13.3
+PackageLocale: en-US
+Publisher: WhiteMagic
```
Comment (ItzLevvie) [inline manifests/w/WhiteMagic/JoystickGremlin/13.3/WhiteMagic.JoystickGremlin.locale.en-US.yaml:6]:
```suggestion
Publisher: H2IK
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by iskela45** [comment]
nvm, did another search and found anoter

**Pull request closed without merging**